include/crm/cluster/internal.h

```diff
@@ -93,6 +89,12 @@ typedef struct pcmk__node_status {
 
     time_t when_member;         // Since when node has been a cluster member
     time_t when_online;         // Since when peer has been online in CPG
+
+    /* @TODO The following are currently needed only by the Corosync stack.
```

Comment:
Turns out `cluster_layer_id` is used in the controller and attribute manager too

Reply:
xml_id would probably be more appropriate for those uses (they're the same for Corosync nodes, just string vs int, so it wouldn't matter currently)

Reply:
agreed, slightly less efficient but not significant